kloppy/infra/serializers/event/wyscout/deserializer_v3.py

```diff
@@ -184,12 +183,10 @@ def _parse_foul(raw_event: Dict) -> Dict:
 def _parse_card(raw_event: Dict) -> Dict:
     qualifiers = _generic_qualifiers(raw_event)
     card_type = None
-    if _has_tag(raw_event, wyscout_tags.RED_CARD):
-        card_type = CardType.RED
-    elif _has_tag(raw_event, wyscout_tags.YELLOW_CARD):
+    if _check_secondary_event_types(raw_event, ["yellow_card"]):
```

Comment:
What happens when a player gets a second yellow card?

Reply:
Yes, all clear to me now. I had expected that they would differentiate between second yellow cards and straight red cards. Thanks!